src/eradiate/scenes/integrators/_core.py

```diff
@@ -64,3 +64,29 @@ class Integrator(NodeSceneElement, ABC):
         init_type="bool, optional",
         default="False",
     )
+
+    stokes: bool = documented(
+        attrs.field(
+            default=False,
+            converter=attrs.converters.optional(bool),
+        ),
+        doc="If true, calculates and outputs the polarized components of"
+        "the stokes vector. Note that this will only work with a polarized"
+        "mode and a sensor that measures radiance.",
+        type="bool",
+        init_type="bool, optional",
+        default="False",
+    )
+
+    meridian_align: bool = documented(
+        attrs.field(
+            default=True,
+            converter=attrs.converters.optional(bool),
+        ),
+        doc="If true, align the stokes vector to the meridian plane. Otherwise"
+        "default to aligning to the sensor's x-axis. This parameter is only"
+        "relevant in polarized mode, when measuring radiance with Stokes=True.",
```

Comment:
```suggestion
        doc="If ``True``, align the stokes vector to the meridian plane. Otherwise, "
        "default to aligning to the sensor's x-axis. This parameter is only "
        "relevant in polarized mode, when measuring radiance with ``stokes=True``.",
```